functions/Parse-DbaCsvMetadata.ps1

```diff
@@ -0,0 +1,360 @@
+﻿function Get-UserPrompt {
+    param (
+        [string]$msg = "Would you like to continue?"
+    ) 
+    $choice = ""
+    while ($choice -notmatch "[y|n]"){
+        $choice = Read-Host "$msg (Y/N)"
+        }
+    
+    if ($choice -eq "N"){
+        Write-Warning "Good choice you made."
+        exit
+    }
+    else 
+    {
+        Write-Warning "Continuing... Waaaaaa (╯°□°）╯︵ ┻━┻)"
+    } 
+}
+  
+
+function Parse-DbaCsvMetaData {
+    <#
+
+    .SYNOPSIS 
+
+    Simple template
+
+
+
+    .DESCRIPTION
+
+    By default, all SQL Agent categories for Jobs, Operators and Alerts are copied.  
+
+
+
+    .PARAMETER Csv
+
+    CSV file to be processed. (Single, multiple file, if left blank dialog prompts for selection)
+
+
+
+    .PARAMETER HeadersInRow
+
+    The rownumber where the header is located. (0 means no header)
+
+
+
+    .PARAMETER SpeedTest
+
+    If set, it will read the entire file and outputs read / time statistics.
+
+
+    .NOTES 
+
+    Original Author: You (@YourTwitter, Yourblog.net)
+
+

```

Comment:
This is the right stuff :) But the paste somehow added way more line breaks than necessary to this whole section, starting from SYNOPSIS. Please condense.